frontend/public/locales/fr/learn/deciding-when-to-start-your-public-pensions.json

```diff
@@ -48,12 +48,12 @@
       ]
     },
     "description": "Le tableau montre l'évolution des montants mensuels de votre pension de retraite du RPC en fonction de l'âge auquel vous commencez à la recevoir. Il montre que plus vous attendez pour commencer à recevoir votre pension, plus vous percevrez d'argent. Votre pension de retraite du RPC pourrait plus que doubler si vous commencez à la recevoir à 70 ans plutôt qu'à 60 ans. Le tableau illustre l'augmentation de la pension en fonction de l'âge et utilise un montant de pension illustratif.",
-    "disclaimer": "<strong>Avertissement\u00a0:</strong> Ce site Web vise à vous donner des conseils sur le bon moment pour recevoir vos pensions publique. Nous ne fournissons pas de conseils financiers. Une fois que vous aurez toute l'information dont vous avez besoin, nous vous encourageons à demander l'aide d'un conseiller financier.",
+    "disclaimer": "<strong>Clause de non-responsabilité\u00a0:</strong> Ce site Web vise à vous donner des conseils sur le bon moment pour recevoir vos pensions publique. Nous ne fournissons pas de conseils financiers. Une fois que vous aurez toute l'information dont vous avez besoin, nous vous encourageons à demander l'aide d'un conseiller financier.",
     "heading": "Pension de retraite du RPC",
-    "p1": "Quel que soit le nombre de fois où vous avez changé d'emploi, en travaillant au Canada, vous avez contribuer au Régime de pensions du Canada (RPC) ou au <a1>Régime de rentes du Québec (RRQ).</a1> Le montant de votre pension de retraite du RPC dépend du montant et de la durée de vos contributions. Le RPC vous permet de choisir la date de début de votre pension de retraite, et chaque mois que vous reportez augmente votre montant mensuel. Le montant mensuel le plus élevé que vous puissiez recevoir est atteint à l'âge de 70 ans, après quoi il n'y a plus d'avantage à attendre. Si vous avez besoin d'argent plus tôt, vous pouvez commencer à recevoir votre pension dès l'âge de 60 ans, mais avec une réduction permanente.",
+    "p1": "Quel que soit le nombre de fois où vous avez changé d'emploi, en travaillant au Canada, vous avez contribuer au Régime de pensions du Canada (RPC) ou au <a1>Régime de rentes du Québec (RRQ).</a1> Le montant de votre pension de retraite du RPC dépend du montant et de la durée de vos contributions. Le RPC vous cotisations de choisir la date de début de votre pension de retraite, et chaque mois que vous reportez augmente votre montant mensuel. Le montant mensuel le plus élevé que vous puissiez recevoir est atteint à l'âge de 70 ans, après quoi il n'y a plus d'avantage à attendre. Si vous avez besoin d'argent plus tôt, vous pouvez commencer à recevoir votre pension dès l'âge de 60 ans, mais avec une réduction permanente.",
```

Comment:
To remain consistent with other links in the application, the period should be moved outside of the `<a1>` here..

ex:

```
<a1>Régime de rentes du Québec (RRQ)</a1>.
```

Reply:
Fixed